components/FeaturePreview/styles.module.css

```diff
@@ -0,0 +1,12 @@
+.featurePreview {
+  font-size: calc(var(--ifm-font-size-base) * 0.8);
+  line-height: 24px;
+  border-top: solid 1px var(--ifm-color-emphasis-500);
+  padding: 6px 0.5rem 0 0.5rem;
```

Comment:
Nitpick: units could be unified (f.e. rems only).

Reply:
Done